pkg/virt-handler/virtwrap/manager.go

```diff
@@ -416,3 +427,125 @@ func (l *LibvirtDomainManager) KillVM(vm *v1.VM) error {
 	l.recorder.Event(vm, kubev1.EventTypeNormal, v1.Deleted.String(), "VM undefined")
 	return nil
 }
+
```

Comment:
I would not suggest spending time on making such a layout - this entire approach is fundamentally wrong & cannot be merged. Any VDSM specific logic needs to remain outside KubeVirt and just interact with the features exposed in the k8s API.